src/pages/WordsOverview.js

```diff
@@ -1,6 +1,5 @@
 import { projectFirestore } from "../firebase/config";
 import { useState, useEffect } from "react";
-import Form from "../components/Form";
 
```

Comment:
V useEffect hooku máš console.log(snapshot). Jaký tam má význam, musí tam být?
Na conosle.log obecně pozor. Je dobrý když něco zkoušíš, testuješ, prostě potřebuješ vidět když to vyvíjíš, ale ve finální verzi kódu by se neměl objevovat pokud nemá praktivký význam.
A sorry, že jsem ten koment dal víš, ale nejde mi dá koment přímo na ten console.log...

Reply:
Jakej význam má rádek 22, proč je potřeba nastavit error na prázdný string? Má přece defaultní hodnotu.